docs/release.rst

```diff
@@ -9,7 +9,7 @@ In that case just ignore all section that require you to be an Anaconda maintain
 0. prerequisites
 
 - you need an up to date anaconda source code checkout
-- if there is an active Fedora devel branch, merge it with ``git merge --no-ff f<version>-devel``
+- if there is an active Fedora release branch, merge it with ``git merge --no-ff f<version>-release``
```

Comment:
Shouldn't we completely remove this line?

Reply:
it's pretty confusing to me so I'm for dropping it.

Reply:
Oh, looking at it now that's indeed redundant, lets remove it. :)